extensions/http-curl/tests/HTTPIntegrationBase.h

```diff
@@ -91,4 +87,74 @@ void CoapIntegrationBase::setUrl(std::string url, CivetHandler *handler) {
   }
 }
 
+class VerifyC2Base : public CoapIntegrationBase {
+ public:
+  explicit VerifyC2Base(bool isSecure)
+      : isSecure(isSecure) {
+    char format[] = "/tmp/ssth.XXXXXX";
+    dir = testController.createTempDirectory(format);
+  }
+
+  virtual void testSetup() {
+    LogTestController::getInstance().setDebug<utils::HTTPClient>();
+    LogTestController::getInstance().setDebug<LogTestController>();
+    std::fstream file;
+    ss << dir << "/" << "tstFile.ext";
```

Comment:
For C2 verification I think we don't need test files. Or do I miss something?

Reply:
yes. We don't need them. Was a remnant from previous code. Removed it.